src/Util/Asserts.h

```diff
@@ -0,0 +1,30 @@
+//
+//  Asserts.h
+//
+//  Copyright (c) 2016 Open Whisper Systems. All rights reserved.
+//
+
```

Comment:
ci failure:
❌  /Users/travis/build/WhisperSystems/SignalServiceKit/src/Network/OWSCensorshipConfiguration.m:15:5: implicit declaration of function 'OWSAssert' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
    OWSAssert(e164PhonNumber.length > 0);

Note that tests run in the context of `SignalServiceKit/Example/TSKitiOSTestApp/`